lymph/node.py

```diff
@@ -14,7 +15,7 @@ class AbstractNode:
     def __init__(
         self,
         name: str,
-        state: int,
+        state: int = 0,

```

Comment:
While not an issue, I wouldn't add default arguments in an abstract base class. Because it should not be created by the user anyways, but only via the `super()` calls from the inheriting classes. So, if there is an issue with the initialization of a `Tumor` or `LymphNodeLevel` instance, then their respective `__init__` methods should handle that.